sympy/physics/control/lti.py

```diff
@@ -546,6 +546,123 @@ def from_rational_expression(cls, expr, var=None):
             raise ZeroDivisionError("TransferFunction cannot have a zero denominator.")
         return cls(_num, _den, var)
 
+    @classmethod
+    def from_coeff_lists(cls, num_list, den_list, var):
+        r"""
+        Creates a new ``TransferFunction`` efficiently from a list of coefficients.
+
+        Parameters
+        ==========
+
+        num_list : List
+            List comprising of numerator coefficients.
+        den_list : List
+            List comprising of denominator coefficients.
+        var : Symbol
+            Complex variable of the Laplace transform used by the
+            polynomials of the transfer function.
+
+        Raises
+        ======
+
+        ValueError
+            When ``num_list`` or ``den_list`` are not of data type `list`.
```

Comment:
This should probably work with any sequence or iterable of sympifiable elements.

Reply:
Okay, we don't need to enforce list here. I have removed the `isinstance` check. Yet, I have made the documentation have all examples with lists only just so that it is consistent for viewers.

Reply:
I think that it is technically precise to be ‘Sequence’ though.
Iterable cannot have len or support getitem

https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes

Reply:
Yes, you are right. Thanks for the documentation. I'll replace `Iterable `with `Sequence`.